package.json

```diff
@@ -101,6 +97,7 @@
     "jest": "^24.8.0",
     "jest-styled-components": "^6.3.1",
     "jsx-to-string": "^1.4.0",
+    "kind-of": "^6.0.3",
```

Comment:
Hmm, I'm not sure if we need this to resolve the alert. Since after `yarn upgrade`, there is no error when `yarn audit`.

Reply:
Removed, thanks!